### DISCUSSION TITLE
Does the flux_function in PorousFlowSink accept Volumetric flux?

### DISCUSSION THREAD

**Discussion by batodon** [idaholab/moose#22798] [Q&A General] [answered]
Hello,

Every PorousFlow tutorial I see made use of the mass flux and no mention of volumetric flux. In tutorial08, if we were to use water as the fluid being injected into the media, and hence a volumetric flux of 1e-5m/s, will the flux_function in the input file accept this value as a volumetric flux or it will be registered as a mass flux? In other words, do we strictly have to input a mass flux regardless of the fluid we use? Thanks.

**Comment by WilkAndy** [comment, marked as answer]
Usually it will be a mass flux.  However, it will be a volume flux if you use `multiply_by_density = false` in your Kernels, or in https://mooseframework.inl.gov/source/actions/PorousFlowBasicTHM.html or in https://mooseframework.inl.gov/source/actions/PorousFlowFullySaturated.html .   Please see the latter for some remarks (most particularly, the steady-state Laplace eqn, if that is pertinent to you).

The `multiply_by_density` flag also has some documentation: "If true, then the Kernels for fluid flow are multiplied by the fluid density. If false, this multiplication is not performed, which means the problem becomes more linear, but care must be taken when using other PorousFlow objects, since MOOSE will be computing volume fluxes, not mass fluxes."

**Reply by Traiwit** [reply]
@WilkAndy so the flux unit will be m3 time-1 instead of kg m-2 time-1 ?

**Reply by WilkAndy** [reply]
No, the volumeric flux unit will be m3 m-2 time-1 = m time-1

**Reply by WilkAndy** [reply]
Did the above explanation answer your question, @batodon ?   (If so, please mark as "answered" :-)

**Reply by batodon** [reply]
Thanks @WilkAndy . Really appreciate. It was helpful!
